toolkit/tools/scheduler/scheduler.go

```diff
@@ -189,6 +192,20 @@ func main() {
 	if err != nil {
 		logger.Log.Fatalf("Unable to build package graph.\nFor details see the build summary section above.\nError: %s.", err)
 	}
+
+	if *useCcache {
+		logger.Log.Infof("  ccache is enabled. processing created artifacts under (%s)...", *ccacheDir)
+		var ccacheManager ccachemanagerpkg.CCacheManager
+		err = ccacheManager.Initialize(*ccacheDir, *ccacheConfig)
+		if err == nil {
+			err = ccacheManager.UploadAllPkgGroupCCaches()
+			if err != nil {
+				logger.Log.Warnf("Failed to archive CCache artifacts. Error: %s.", err)
```

Comment:
Most other errors are printed as %v - should this be as well?

Reply:
Fixed.